src/axom/spin/SpatialOctree.hpp

```diff
@@ -29,18 +29,18 @@ class SpatialOctree : public OctreeBase<DIM, BlockDataType>
 {
 public:
 
-  typedef primal::BoundingBox<double,DIM> GeometricBoundingBox;
-  typedef primal::Point<double,DIM> SpacePt;
-  typedef primal::Vector<double,DIM> SpaceVector;
+  using GeometricBoundingBox = BoundingBox<double,DIM> ;
+  using SpacePt = Point<double,DIM> ;
+  using SpaceVector = Vector<double,DIM> ;
```

Comment:
So, ``SpacePt`` is now an alias to ``spin::Point<double,DIM>``, which is an alias to ``primal::Point< double, DIM>.

All this aliasing can get very confusing, which makes it harder to grok and follow the code. I don't see a strong technical reason to justify all this indirection. It would be better to map aliases directly to the actual type.

Reply:
see previous; resolved